Underscore.cs/List/Implementation/Partition.cs

```diff
@@ -113,6 +113,14 @@ public IEnumerable<IEnumerable<T>> Chunk<T>( IList<T> list, Func<T, bool> on )
             return Tuple.Create( new T[ ] { } as IEnumerable<T>, new List<T>( list ) as IEnumerable<T>);
         }
 
+        /// <summary>
+        /// Splits enumerable into two seperate collections based off of the passed func, all items passing the condition 
+        /// are placed in the first item in the tuple and the others placed in the second item in the tuple
+        /// </summary>
+        /// <typeparam name="T">Type of the items in the enumerable</typeparam>
```

Comment:
i don't really think type param comments are necessary when they're this obvious